new/LIT6715PPQwerdat.xml

```diff
@@ -0,0 +1,70 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6715PPQwerdat" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Protective prayer against <foreign xml:lang="gez">qʷərḍat</foreign>, containing <foreign xml:lang="gez">ʾasmāt</foreign></title>
```

Comment:
I am just wondering whether we may use - for the benefit of the reader - the English term in the title, perhaps with the Ethiopic term in brackets, e.g. `Protective prayer against stomachache (<foreign xml:lang="gez">qʷərḍat</foreign>), containing ...`